simd/2021/SIMD-10-01.md

```diff
@@ -0,0 +1,235 @@
+
+AK: for RPCs the future use case, there are 2 worlds, fpenv is a spec construct because we want formal, leave it in the spec without going into the implementation, another world is where there is a practical manifestation of spec langauge in the future, like the RPC, but we should have a sketch of how it would work in that use case. fpenv seems like a compromise, or at least not clear how rpc would work in the current spec. Drafting that will be helpful. If it doesn't work, we should make the construct stronger, or keep it to the spec language.
+
+DeG: wait for Dan to write something up, then we can come back to this.
+
+AK: can we also close some issues, multiple things 
+
+ZA: AI to consolidate the issues. 
+
+### Smaller groups of SIMD instructions (like AES)
+
+DeG: if we are introducing extensions to SIMD, we can move to design to see if we can move this through, i would like to see use case, real applications using it, driven by use cases. One use case for 64x2 is that it's hard to do some things without them. Performance is another aspect. Can have some design consensus within this group, before presenting to CG. We can streamline and have multiple instructions, but still need strong use cases.
+
+PP: we want to do the use cases in this group? then surface it to the CG/ move to main spec.
+
+DeG: If someone is interested in adding an extension, we should document it ahead of time, get a first pass from this group.
+
+PP: if someone has a SIMD instruction to explore, they can come to us first
+
+DeG: what do you have in mind for AES? what are you looking to get out of this?
+
+PP: I’m trying to find what else we can use AES for? From instructions perspective, relatively straightforward to implement..
+
+LH: issue 31 about AES, decided it is not for relaxed-simd, suggested it would be spun off as its own proposal
+
+
+ZA: We haven’t really had small proposals about testing how fast we can get through the process. This is one example we can try if we want to streamlined. We need use cases - PP are you interested in driving this forward? 
+
+PP: Will follow up on the issue.
+
+ZA: Out of scope for relaxed SIMD, but in scope for the subgroup as a SIMD adjacent proposal.
+
+
+### [i32x4.trunc_sat_f32x4_u](https://github.com/WebAssembly/relaxed-simd/issues/21#issuecomment-929425797)
+
+ZA: Relaxed version for the unsigned instructions is the only one in the current proposal that is 8 instructions long, still better than the non-relaxed version.
+
+MD: we never had a formal req for 1 instruction lowering
+
+ZA: Anyone disagrees/agrees? 
+
+LH: no objections
+
+<No objections> 
+
+### [wasi-parallel](https://github.com/WebAssembly/wasi-parallel)
+
+PP: Announcement about WASI parallel: officially published the parallel proposal for WASI, it’s now at phase 2 in the WASI process. The idea behind this is that you have a parallel construct that can be used to run parts of the module in parallel, like how you would use OpenMP. 
```

Comment:
I made a mistake, it is phase 1, not phase 2.

Reply:
Fixed, thanks for confirming!